### PR TITLE
Update outdated jq link

### PR DESCRIPTION
**Pull request by steve-hardman**
As requested updated outdated links to `jq`

This will fix https://github.com/kubernetes/website/issues/43285

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ You will need to have the following tools installed:
 
 - `cosign` ([install guide](https://docs.sigstore.dev/cosign/installation/))
 - `curl` (often provided by your operating system)
-- `jq` ([download jq](https://stedolan.github.io/jq/download/))
+- `jq` ([download jq](https://jqlang.github.io/jq/download/)
```
Comment (dipesh-rawat) [inline content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md:18]: Minor fix needed.
```suggestion
- `jq` ([download jq](https://jqlang.github.io/jq/download/))
```

**Review by dipesh-rawat — COMMENTED** [review]
@steve-hardman Thanks for your contribution. Could you please verify if there are additional files in English with outdated links and update them too? You might want to consider searching for `https://stedolan.github.io/` in the website repo to identify the files requiring updates.

**Comment by steve-hardman** [comment]
Ok aplogies found few more files that need updating; on it now. Thanks @dipesh-rawat for the nudge!

**Comment by steve-hardman** [comment]
Before i proceed can I know from any of the maintainers what do with links in old blog posts like 2021 & 2020 should i leave them as is or update them.

Also, I see the old link https://stedolan.github.io/jq/ autmaotically redirects to new link https://jqlang.github.io/jq/ so we are okay potentally without these changes. Not sure though.

**Comment by utkarsh-singh1** [comment]
All build error seems to point to the same issue --> `Build exceeded maximum allowed runtime`. Seems like Netlify bug, try to push one more commit with updated changes and files.

**Comment by utkarsh-singh1** [comment]
> On a separate note: Before i proceed can I know from any of the maintainers what do with links in old blog posts like 2021 & 2020 should i leave them as is or update them.
> 
> Also, I see the old link https://stedolan.github.io/jq/ autmaotically redirects to new link https://jqlang.github.io/jq/ so we are okay potentally without these changes. Not sure though.

1. It would be better to update them, changes to blog posts more than 3 years are not reviewed on priority.
2. We don't know for how much time redirect will work, it would be better to change with current link.

**Comment by steve-hardman** [comment]
> > On a separate note: Before i proceed can I know from any of the maintainers what do with links in old blog posts like 2021 & 2020 should i leave them as is or update them.
> > Also, I see the old link https://stedolan.github.io/jq/ autmaotically redirects to new link https://jqlang.github.io/jq/ so we are okay potentally without these changes. Not sure though.
> 
>     1. It would be better to update them, changes to blog posts more than 3 years are not reviewed on priority.
> 
>     2. We don't know for how much time redirect will work, it would be better to change with current link.

Thanks @utkarsh-singh1 Uupdated all the files now.

**Comment by sftim** [comment]
Thanks

/lgtm

**Comment by SaranBalaji90** [comment]
@steve-hardman Thanks for helping with the issue. There is a typo [here](https://github.com/kubernetes/website/blob/6b1a95ff541a1fca004d72fdac6c58b8ff630643/content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md?plain=1#L31) in old link. Can you fix this as well? Thanks.

**Pull request merged**